.github/workflows/json_schemas_pr_check.yaml

```diff
@@ -0,0 +1,58 @@
+name: Check JSON schemas consistency
+
+# The json_schemas package contains TypeScript code that generates JSON schema files.
+# This workflow ensures that any changes to the TypeScript code are reflected in the generated JSON schema files.
+# If there is a mismatch, the workflow fails, prompting the developer to regenerate the JSON schema files.
+
+on:
+    pull_request:
+        # List all .ts - .json file pairs that need to be kept in sync
+        paths:
+            - 'packages/json_schemas/src/actor.schema.ts'
+            - 'packages/json_schemas/schemas/actor.schema.json'
+
+jobs:
+    check_json_schemas:
+        name: 'Check JSON schemas consistency'
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v5
+
+            - name: Use Node.js ${{ matrix.node-version }}
+              uses: actions/setup-node@v5
+              with:
+                node-version: ${{ matrix.node-version }}
+
+            - name: Cache node_modules
+              uses: actions/cache@v4
+              with:
+                path: '**/node_modules'
+                key: ${{ runner.os }}-${{ matrix.node-version }}-modules-${{ hashFiles('**/package-lock.json') }}
```

Comment:
this is likely a copy pasta from other workflow, but here it makes no sense to check it for different node versions (`${{ matrix.node-version }}`)

side note: is that cache action even needed? i believe `setup-node` has native support for caching (and with npm it should actually work fine, unlike with other package managers)